google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/Job.java

```diff
@@ -220,14 +223,18 @@ public boolean isDone() {
     }
     try (Scope isDoneScope = isDone != null ? isDone.makeCurrent() : null) {
       Job job = bigquery.getJob(getJobId(), JobOption.fields(BigQuery.JobField.STATUS));
-      return job == null || JobStatus.State.DONE.equals(job.getStatus().getState());
+      return job == null || job.hasDoneState();
```

Comment:
@PhongChuong @jinseopkim0 

Do either of you recall the logic behind returning true if the job is not found (`job == null`)?